integration_test_lite.py

```diff
@@ -65,29 +65,32 @@ def basic_request(self, url, params, needs_access_token, verb="get"):
         :returns: None if the response was ok. Otherwise, an error object
         :rtype: dict
         """
-        query_params = endpoint["query_params"]
+        query_params = endpoint['query_params']
         # Add random query parameter with random value to bypass caching
         query_params[uuid.uuid4().hex] = uuid.uuid4().hex
 
-        request = await self.basic_request(
-            endpoint["base_url"],
+        response = await self.basic_request(
+            endpoint['base_url'],
             query_params,
-            endpoint["needs_access_token"]
+            endpoint['needs_access_token']
         )
-        response_code = request.status
+        response_code = response.status
 
         allowed_response_codes = [200]
-        if 'allow_400' in endpoint and endpoint['allow_400']:
+        if endpoint.get('allow_400'):
             allowed_response_codes.append(400)
 
         if response_code not in allowed_response_codes:
             api_info = endpoint
-            api_info["response_code"] = response_code
+            api_info['response_code'] = response_code
 
             try:
-                api_info["response_body"] = await request.json()
-            except ValueError as error:
-                api_info["response_body"] = str(error)
+                api_info['response_body'] = await response.json()
```

Comment:
I think you don't need to `await` here again (and also in except block) since you've awaited the response object at line 72.